pennylane/labs/resource_estimation/ops/qubit/parametric_ops_single_qubit.py

```diff
@@ -295,7 +295,7 @@ def default_resource_decomp(cls, eps=None, **kwargs) -> list[GateCount]:
             .. math:: T_{count} = \lceil(1.149 * log_{2}(\frac{1}{\epsilon}) + 9.2)\rceil
 
         """
-        eps = eps or kwargs["config"]["error_rx"]
+        eps = eps or kwargs["error_rx"]
```

Comment:
In that case, the configuration shouldn't have `"error_rx"`, it should just have `RX: {'eps': ...}`, so that when it's passed into this function, it takes the place of `eps` directly.

Reply:
Yep. I'm trying to just get the tests to pass for now, but I will rename things accordingly.